src/next-reminders/lambda/lambda.ts

```diff
@@ -0,0 +1,46 @@
+import * as AWS from 'aws-sdk';
+import * as SSM from 'aws-sdk/clients/ssm';
+import { createObjectCsvStringifier } from 'csv-writer';
+import { createDatabaseConnectionPool } from '../../lib/db';
+import { getDatabaseParamsFromSSM } from '../../lib/ssm';
+import { getNextReminders } from '../lib/db';
+import { getCurrentReminderPeriod } from '../lib/utils';
+
+const ssm: SSM = new AWS.SSM({ region: 'eu-west-1' });
+const s3 = new AWS.S3();
+
+const currentReminderPeriod = getCurrentReminderPeriod();
+const S3_KEY = `next-reminders/reminder_period=${currentReminderPeriod}/next-reminders.csv`;
```

Comment:
Lauren just pointed out that this can just be:
`ophan-raw-support-reminders/next-reminders/next-reminders.csv`
Because it's a full refresh table.

The signups tables will still need the partition key.

Reply:
oh of course! makes sense!